core/include/native_sched.h

```diff
@@ -30,6 +30,7 @@ extern "C" {
 #ifdef CPU_NATIVE
 #include <stdio.h>
 
+#if __GLIBC__
```

Comment:
Not a blocker for this PR but why is there no `#elseif __musl__ ...`?

Reply:
In musl those are already provided (causing the conflict).